store/playbackManager.ts

```diff
@@ -120,6 +120,13 @@ export const getters: GetterTree<PlaybackManagerState, PlaybackManagerState> = {
       return state.queue?.[state.currentItemIndex]?.MediaType;
     }
     return null;
+  },
+  getCurrentTime: (state) => {
```

Comment:
Getters should perform some kind of processing on the state to return a different value (For example filtering data or merging two stores' data).

This just returns the state as-is, so it's not necessary. You can just access the state directly in these cases.